### ISSUE TITLE
iocage fails after iocage fetch FreeBSD 12

### ISSUE THREAD

**Issue by java4devgmail** [iocage/iocage#839]
Make sure to follow and check these boxes before submitting an issue! Thank you.

- [x] Supply `iocage --version`
iocage --version
Version 1.1 RELEASE 2019/01

- [x] Supply the commands used, along with any steps to recreate it.
On a fresh installed FreeBSD 12
I did 
freebsd-update
pkg install py36-iocage
iocage activate zroot
iocage fetch 
[8] 12.0-RELEASE

then followign fails
iocage list -PR

I even tried via the ports
cd /usr/ports/sysutils/iocage
#make deinstall
#make install clean

- [x] Provide the output from the command you issued.
 # iocage list -PR
********************************************************************************
fdescfs(5) is not mounted, performance may suffer. Please run:
mount -t fdescfs null /dev/fd
You can also permanently mount it in /etc/fstab with the following entry:
fdescfs /dev/fd  fdescfs  rw  0  0
********************************************************************************

Branch 12.0-RELEASE does not exist at https://github.com/freenas/iocage-ix-plugins.git!
Using "master" branch for plugin, this may not work with your RELEASE
Traceback (most recent call last):
  File "/usr/local/bin/iocage", line 10, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/iocage_cli/list.py", line 99, in cli
    ioc_common.logit({"level": "INFO", "message": _list})
  File "/usr/local/lib/python3.6/site-packages/iocage_lib/ioc_common.py", line 104, in logit
    _callback(content, exception)
  File "/usr/local/lib/python3.6/site-packages/iocage_lib/ioc_common.py", line 60, in callback
    log.info(message)
  File "/usr/local/lib/python3.6/logging/__init__.py", line 1308, in info
    self._log(INFO, msg, args, **kwargs)
  File "/usr/local/lib/python3.6/logging/__init__.py", line 1444, in _log
    self.handle(record)
  File "/usr/local/lib/python3.6/logging/__init__.py", line 1454, in handle
    self.callHandlers(record)
  File "/usr/local/lib/python3.6/logging/__init__.py", line 1516, in callHandlers
    hdlr.handle(record)
  File "/usr/local/lib/python3.6/logging/__init__.py", line 865, in handle
    self.emit(record)
  File "/usr/local/lib/python3.6/site-packages/iocage_cli/__init__.py", line 76, in emit
    print(log, file=sys.stdout)
UnicodeEncodeError: 'ascii' codec can't encode character '\u2122' in position 1815: ordinal not in range(128)

- [x] Supply what you expected the result or output to be
a list of plugins
- [x] Checked that the problem has not already been fixed on `master` if using
 a stable release.

**Comment by skarekrow** [comment]
Can you paste your `env` and `locale` please?

**Comment by java4devgmail** [comment]
# env
TERM=xterm
BLOCKSIZE=K
MAIL=/var/mail/root
PATH=/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/root/bin
SHELL=/bin/csh
HOME=/root
USER=root
HOSTTYPE=FreeBSD
VENDOR=amd
OSTYPE=FreeBSD
MACHTYPE=x86_64
SHLVL=1
PWD=/root
LOGNAME=root
GROUP=wheel
HOST=---
REMOTEHOST=---
EDITOR=vi
PAGER=less
# locale
LANG=
LC_CTYPE="C"
LC_COLLATE="C"
LC_TIME="C"
LC_NUMERIC="C"
LC_MONETARY="C"
LC_MESSAGES="C"
LC_ALL=

**Comment by skarekrow** [comment]
That's the problem, set LANG to a UTF-8 locale, LC_ALL wouldn't hurt either.

**Comment by java4devgmail** [comment]
I added in the /etc/login.conf in the default profile
```

\
        :umask=022:\
        :charset=UTF-8:\
        :lang=en_US.UTF-8:\
        :setenv=LC_ALL="en_US.UTF-8":

 # cap_mkdb /etc/login.conf
```



and  iocage now works.

even though  when i do locale LC_ALL is still empty but somehow it affects iocage
```
  # locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_ALL=

```


Thank you.

**Comment by skarekrow** [comment]
@java4devgmail Great to hear! This will be fixed in 1.2 thanks to your issue :)

Thanks!
